ambari-server/src/main/resources/common-services/ZEPPELIN/0.7.0/configuration/zeppelin-config.xml

```diff
@@ -77,7 +77,7 @@
   </property>
   <property>
     <name>zeppelin.notebook.storage</name>
-    <value>org.apache.zeppelin.notebook.repo.FileSystemNotebookRepo</value>
+    <value>org.apache.zeppelin.notebook.repo.VFSNotebookRepo</value>
```

Comment:
otherwise upgrade to HDP-2.6.1/2.6.2 would overwrite the value.
During install, stack advisor will provide the right value (other jira by @vbrodetskyi )